Changelog.md

```diff
@@ -5,7 +5,7 @@ Language Features:
 Compiler Features:
 
 Bugfixes:
-
+Codegen: Allow emitting events via module member access
```

Comment:
```suggestion
Codegen: Allow emitting events via module member access.
```

Reply:
Also, leave the whitespace alone :)

Hint: if you see any empty red or green lines in your changelog diff, you're doing it wrong :P